### DISCUSSION TITLE
When navigating between two different layouts, the design breaks.

### DISCUSSION THREAD

**Discussion by mdashikulislam** [livewire/livewire#8977] [1. Help]
[screen-capture (22).webm](https://github.com/user-attachments/assets/7c9968c7-108a-4981-a8e5-32a29c8eb8f5)

I am using different template for frontend and admin panel. but when i `wire:navigate` from frontend to admin panel or from admin panel to frontend-end my design is broken. but if i reload page than design should be ok. issue is occur when i using  wire:navigate.

**Comment by otavio-araujo** [comment]
Did you fixed it?

I get a similar problem, but, its the scrollbarr that just do not render when i use wire:navigate.

If you fixed? what did you do?

**Reply by mdashikulislam** [reply]
still not fixed. I am doing if layout change that time i reload page.

**Reply by danie-ramdhani** [reply]
i think it because you use different css inside `head` tag for that layout and because `wire:navigate` will loaded once like script (https://livewire.laravel.com/docs/navigate#scripts-in-head-are-loaded-once), it will override some of you style. (same scenario when you navigate to laravel error page like 404)

maybe workaround is load your css file inside body tag?
